src/hash.rs

```diff
@@ -1,3 +1,21 @@
+/*
+ * Copyright 2019 Bitwise IO, Inc.
+ * Copyright 2019 Cargill Incorporated
```

Comment:
Looking at the history the license should only be for Bitwise, We only add Cargill if there was a significant change to the file.

Reply:
Fixed